BucketInfoModelVersion.md

```diff
@@ -0,0 +1,82 @@
+# BucketInfo Model Version History
```

Comment:
Can you create a "documentation" directory and put this file in there? Just so it looks more tidy :broom: 

Or put it in lib/models/ directly, close to the code, e.g. I already see lib/storage/metadata/mongoclient/Mongoclient.md.